### PR TITLE
kata-webhook: Update README.md

### PR DESCRIPTION
**Pull request by fidencio**
The whole vendor directory has to be copied under the kata-webhook
directoy, otherwise we'll face failures such as:
```
fidencio@laerte ~/go/src/github.com/kata-containers/tests/kata-webhook $ docker build -t katadocker/kata-webhook-example:latest .
Sending build context to Docker daemon  13.82kB
Step 1/7 : FROM golang:latest AS builder
 ---> 2421885b04da
Step 2/7 : WORKDIR /go/src/kata-pod-annotate
 ---> Using cache
 ---> 2fbde962f238
Step 3/7 : COPY . ./
 ---> Using cache
 ---> 20cf32a11a54
Step 4/7 : RUN CGO_ENABLED=0 go build -o /go/bin/kata-pod-annotate
 ---> Running in a90ab84e36bc
main.go:18:2: cannot find package "github.com/slok/kubewebhook/pkg/http" in any of:
	/usr/local/go/src/github.com/slok/kubewebhook/pkg/http (from $GOROOT)
	/go/src/github.com/slok/kubewebhook/pkg/http (from $GOPATH)
main.go:19:2: cannot find package "github.com/slok/kubewebhook/pkg/log" in any of:
	/usr/local/go/src/github.com/slok/kubewebhook/pkg/log (from $GOROOT)
	/go/src/github.com/slok/kubewebhook/pkg/log (from $GOPATH)
main.go:20:2: cannot find package "github.com/slok/kubewebhook/pkg/webhook/context" in any of:
	/usr/local/go/src/github.com/slok/kubewebhook/pkg/webhook/context (from $GOROOT)
	/go/src/github.com/slok/kubewebhook/pkg/webhook/context (from $GOPATH)
main.go:21:2: cannot find package "github.com/slok/kubewebhook/pkg/webhook/mutating" in any of:
	/usr/local/go/src/github.com/slok/kubewebhook/pkg/webhook/mutating (from $GOROOT)
	/go/src/github.com/slok/kubewebhook/pkg/webhook/mutating (from $GOPATH)
main.go:15:2: cannot find package "k8s.io/api/core/v1" in any of:
	/usr/local/go/src/k8s.io/api/core/v1 (from $GOROOT)
	/go/src/k8s.io/api/core/v1 (from $GOPATH)
main.go:16:2: cannot find package "k8s.io/apimachinery/pkg/apis/meta/v1" in any of:
	/usr/local/go/src/k8s.io/apimachinery/pkg/apis/meta/v1 (from $GOROOT)
	/go/src/k8s.io/apimachinery/pkg/apis/meta/v1 (from $GOPATH)
The command '/bin/sh -c CGO_ENABLED=0 go build -o /go/bin/kata-pod-annotate' returned a non-zero code: 1
```

Fixes: #2479

Signed-off-by: Fabiano Fidêncio <fidencio@redhat.com>

### CODE REVIEW

File: kata-webhook/README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Kubernetes YAML files required to instantiate the admission
 controller.
 
 ```bash
+$ cp -a ../vendor .
```
Comment (devimc) [inline kata-webhook/README.md:13]: I wonder how expensive is this `cp` and if it could be replaced with a symbolic link (ln -s)

Comment (fidencio) [reply kata-webhook/README.md:13]: Docker will **not** use files outside of the current build directory.
That's the reason we cannot do a `ln -s` ... or even do the copy inside the container.

Comment (devimc) [reply kata-webhook/README.md:13]: yeah - make sense, thanks

**Comment by grahamwhaley** [comment]
A whole bunch of CI fails, but I'm not sure they are related. I just peeked at the 18.04 one:

```
22:09:21 + '[' -n /usr/local/bin/kata-runtime.backup ']'
22:09:21 + '[' -n /usr/local/bin/kata-runtime ']'
22:09:21 + sudo mv /usr/local/bin/kata-runtime.backup /usr/local/bin/kata-runtime
22:09:21 bash -f ./functional/debug_console/run.sh
22:09:21 INFO: Skip: initrd path is empty
22:09:21 Error: No such container: debugconsole
22:09:21 bash .ci/install_bats.sh
22:09:21 /usr/bin/bats
22:09:21 cd integration/docker-compose && \
22:09:21 bats docker-compose.bats
22:09:21 1..1
22:09:21 # bats warning: Executed 0 instead of expected 1 tests
22:09:21 Makefile:215: recipe for target 'docker-compose' failed
22:09:21 make: *** [docker-compose] Error 1
22:09:21 Failed at 86: sudo -E PATH="$PATH" bash -c "make test"
```

**Comment by chavafg** [comment]
/test-ubuntu

ci failures caused because of changes in bats repo, but it has been fixed in #2485

**Comment by chavafg** [comment]
merging this one, the failures above are not related and were already fixed.

**Pull request merged**